### PR TITLE
Additional changes to build 2.6.3 from source

### PR DESCRIPTION
**Pull request by markxnelson**

### CODE REVIEW

**Review by stevshap — APPROVED** [review]

File: package/entrypoint.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ if [ -e /var/lib/rancher/etcd ] && [ ! -e /var/lib/rancher/k3s/server/db/etcd ];
   echo -n 'default' > /var/lib/rancher/k3s/server/db/etcd/name
 fi
 if [ -e /var/lib/rancher/k3s/server/db/etcd ]; then
-  k3s server --cluster-init --cluster-reset &> ./k3s-cluster-reset.log
```
Comment (mgianatagh) [inline package/entrypoint.sh:21]: Update copyright?

File: package/jailer.sh [hunk 2]
```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
+
+# Copyright (C) 2020, 2021, Oracle and/or its affiliates.
```
Comment (mgianatagh) [inline package/jailer.sh:3]: should copyright be 2020, 2022?

File: Dockerfile.dapper [hunk 3]
```diff
@@ -24,7 +24,8 @@ RUN yum-config-manager --enable ol7_optional_latest && \
 RUN yum update -y && \
     # software collections repo needed for git 2.x on OL7
     yum-config-manager --add-repo=http://yum.oracle.com/repo/OracleLinux/OL7/SoftwareCollections/x86_64 && \
-    yum install -y bash rh-git227 gcc glibc-static docker-cli vim less libffi file curl wget ca-certificates unzip tar jq && \
+    yum install -y bash rh-git227 gcc glibc-static docker-cli vim less libffi file curl wget ca-certificates unzip tar jq \
```
Comment (mgianatagh) [inline Dockerfile.dapper:27]: Should this line end with "&&" and the next line not have that?

Comment (kminder) [reply Dockerfile.dapper:27]: no

File: package/entrypoint.sh (hunk 1 above)
Comment (markxnelson) [reply package/entrypoint.sh:21]: fixed

File: package/jailer.sh (hunk 2 above)
Comment (markxnelson) [reply package/jailer.sh:3]: removed (did not need to customize this file)

**Pull request merged**